projects/graph/src/graph.py

```diff
@@ -98,3 +98,40 @@ def dft(self, starting_vertex_id):
                 # Then, put all of it's children into the stack
                 for neighbor in self.vertices[v]:
                     s.push(neighbor)
+
+    def dft_recursion(self, starting_vert, path=[]):
+        path += [starting_vert]
```

Comment:
Good use of path here, another option is using cache for brief memory storage of vertices visited.